deploy/crds/noobaa.io_backingstores.yaml

```diff
@@ -64,6 +67,7 @@ spec:
                           secret name must be unique.
                         type: string
                     type: object
+                    x-kubernetes-map-type: atomic
```

Comment:
Make sure this should not affect our current behavior.

Reply:
As per below description and default generation, going with atomic type

```
//+mapType | x-kubernetes-map-type | atomic/granular | Applicable to maps. atomic means that the map can only be entirely replaced by a single manager. granular means that the map supports separate managers updating individual fields.
-- | -- | -- | --
```